models/engine/db_storage.py

```diff
@@ -74,3 +74,23 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """retrieve one object based on class and ID,
+        or None if not found"""
+
+        if type(cls) is str:
+            cls = classes.get(cls)
+        if cls not in classes.values():
+            return None
+
+        for key, dbObject in models.storage.all(cls).items():
+            if (str(id) in key):
+                return dbObject
+        return None
+
+    def count(self, cls=None):
```

Comment:
Does this method actually ensure the correct output?? I think it would but it is just calling another method. What if .all() method is returning the incorrect information?? should this method be used to verify all? or am I over thinking?